contracts/contracts/sbtc-registry.clar

```diff
@@ -156,6 +156,32 @@
   )
 )
 
+;; Complete withdrawal request
+(define-public (complete-withdrawal
+    (request-id uint) 
+    (status bool)
+    (bitcoin-txid (optional (buff 32))) 
+    (signer-bitmap (optional uint))
```

Comment:
The design doc have this type as a `(buff 1)`, which I think is too small. So keeping this as `uint` makes sense to me.